src/main/java/edu/KaylaKornelis/advancedjava/Assignment4/StockQuoteApplication.java

```diff
@@ -0,0 +1,193 @@
+package edu.KaylaKornelis.advancedjava.Assignment4;
+
+import edu.KaylaKornelis.advancedjava.Assignment4.Interval;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.List;
+import javax.validation.constraints.NotNull;
+
+/**
+ * This class is used to retrieve stock quotes
+ */
+public class StockQuoteApplication {
+    
+    /**
+     * This main method is the entrance to the application.
+     * @param args an array of command-line arguments for the application
+     */
+    public static void main(String[] args){
+        
+        /**
+         * Call the createListOfStockQuotes method to obtain a List of StockQuote instances. 
+         */
+        System.out.println(createListOfStockQuotes("APPL", "1/1/2014", "2/1/2014"));
+        
+        /**
+         * Call the createListOfStockQuotesWithInterval method to obtain a List of StockQuote instances. 
+         */
+        System.out.println(createListOfStockQuotesWithInterval("APPL", "1/1/2014", "2/1/2014", Interval.WEEKLY));
+        System.out.println("Stocks were obtained for the following interval: " + Interval.WEEKLY);
+        
+        /**
+         * Call the createBasicStock method to obtain the current price for a single StockQuote instance.
+         */
+        System.out.println(createBasicStock("APPL"));
+  
+    }
+    
+    /**
+     * Get a historical list of stock quotes for the provided symbol.
+     * @param symbol the stock symbol to search for
+     * @param from the date of the first stock quote
+     * @param until the date of the last stock quote
+     * @return a list of StockQuote instances
+     * One for each day in the range specified
+     */
+    @NotNull
+    public static List<StockQuote> createListOfStockQuotes(String symbol, String from, String until){
+        /**
+         * Create a new instance of SimpleDateFormat that will be used to 
+         * parse the string arguments to obtain desired start and end dates
+         */
+        SimpleDateFormat sdf = new SimpleDateFormat("MM/dd/YYYY");
```

Comment:
whole word variable make easier to read.

Reply:
fixed